### PR TITLE
[Merged by Bors] - doc: fix doc comment on Nat.geomSum_eq

### PR DESCRIPTION
**Pull request by dwrensha**
The current comment is a copy-paste error from the next lemma `Nat.geomSum_lt`.

### CODE REVIEW

**Review by YaelDillies — COMMENTED** [review]
Would you mind adding a mild warning about nat subtraction and division and say that most likely the lemma the user is after is `geom_sum_mul_add`?

**Comment by dwrensha** [comment]
done in https://github.com/leanprover-community/mathlib4/pull/13106/commits/141feab1fe4203197acf0466b5d80700567a3629

**Review by Ruben-VandeVelde — APPROVED** [review]
Thanks!

maintainer merge

**Comment by riccardobrasca** [comment]
Thanks!

bors merge

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

File: Mathlib/Algebra/GeomSum.lean [hunk 1]
```diff
@@ -589,8 +589,8 @@ end Order
 
 variable {m n : ℕ} {s : Finset ℕ}
 
-/-- If all the elements of a finset of naturals are less than `n`, then the sum of their powers of
-`m ≥ 2` is less than `m ^ n`. -/
+/-- Value of a geometric sum over the naturals. Note: see `geom_sum_mul_add` for a formulation
+that avoids division and subtraction. -/
```
Comment (YaelDillies) [inline Mathlib/Algebra/GeomSum.lean:593]:
```suggestion
that avoids `Nat`-division and `Nat`-subtraction. -/
```

**Pull request closed without merging**